gazebo/rendering/Distortion.cc

```diff
@@ -417,7 +417,8 @@ void Distortion::SetCamera(CameraPtr _camera)
   this->dataPtr->lensDistortionInstance->getTechnique()->getOutputTargetPass()->
       getPass(0)->setMaterial(this->dataPtr->distortionMaterial);
 
-  this->CalculateAndApplyDistortionScale();
+  if (this->dataPtr->legacyMode)
```

Comment:
I just added this fix. Distortion was still being applied when legacyMode was false since this CalculateAndApplyDistortionScale() was always being called. Cropping now seems to behave as intended.